src/style.css

```
@@ -0,0 +1,11 @@
+@import url(./styles/stars.css);
+@import url(./styles/card.css);
+
+html {
+  box-sizing: border-box;
+}
```

It's better to set box-sizing to * selector, because it's not inherited.
```suggestion
* {
  box-sizing: border-box;
}
```

---

done